src/user_templates_api/templates/jupyter_lab/templates/squidpy/metadata.json

```diff
@@ -8,5 +8,13 @@
     ], 
     "is_multi_dataset_template": true,
     "template_format": "jinja",
+    "examples": [
+        {
+            "example_id": 0,
+            "description": "Look at the neturophil cluster of this slide-seq dataset and examine its Ripley's L-function",
```

Comment:
Very minor detail, but can you make sure that descriptions end with periods, especially if they're one sentence.